index.html

```diff
@@ -45,6 +45,7 @@
     <!--/header-->
 
     <!--home section-->
+    <section id="home" class="home">
```

Comment:
Create Separate Sections For Each Page Assign Unique ID For Each Section Keep Class Same As It Division Class